promgen/templates/promgen/project_detail.html

```diff
@@ -40,9 +40,9 @@ <h1>
   </table>
 </div>
 
-<div id="silence-project-{{ project.name|slugify }}" class="panel panel-warning" v-cloak v-show="silenceLabelsProject.has('{{project.name}}')">
+<div id="silence-project-{{ project.name|slugify }}" class="panel panel-warning" v-cloak v-if="silenceLabelsProject.has('{{project.name}}')">
   <div class="panel-heading">
-    <a @click.prevent="toggleTarget('silences-project-{{project.name|slugify}}')" class="btn btn-warning btn-sm" role="button">Silences</a>
+    <a @click="toggleCollapse('silences-project-{{project.name|slugify}}')" class="btn btn-warning btn-sm" role="button">Silences</a>
   </div>
   <table id="silences-project-{{project.name|slugify}}" class="table table-bordered table-condensed">
     <tr v-for="(silence, index) in filterActiveSilences" v-if="silence.labels.project == '{{project.name}}'">
```

Comment:
Using `v-for` with `v-if` is [not recommended](https://v3.vuejs.org/style-guide/#avoid-v-if-with-v-for-essential). Additionally, this iterates over all active alerts/silences, which takes time, but currently doesn't look like a problem. I'll improve this in the next PR.